### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 404 ()

### DISCUSSION THREAD

**Discussion by fanduck-com** [giscus/giscus#1178] [General]
If the discussion is not opened, a 404 error will occur.

If a discussion is opened automatically by opening it manually or writing a comment, the 404 error is resolved.

It doesn't interfere with the function of the website, but can you solve it?

Console log 
>> Failed to load resource: the server responded with a status of 404 ()

**Comment by laymonage** [comment]
It’s intentional. See #552.

**Reply by fanduck-com** [reply]
Oh i see 👍
Thank you so much!
